Week2/num5.js

```diff
@@ -0,0 +1,18 @@
+// 5
+console.log(Math.round(7.25));
+// 5.1
+let z = 7.25 ;
+// 5.2
+console.log(z)
+// 5.3
+let a = Math.round(z)
+// 5.4
+console.log(a)
+// 5.5 
+let highValue ;
+if (a > z) {
+    highValue = a;
+} else { highValue = z}
```

Comment:
I would suggest putting the else on a new line. Like this:

```
if (a > z) {
    highValue = a;
} else { 
    highValue = z;
}
```

Reply:
Besides that it's good. Just a small tip, you can also use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)! But this solution is fine!